front/src/css/todayMain.css

```diff
@@ -0,0 +1,9 @@
+.today-main img {
+    display: block;
+    height: 300px;
+    width: 300px;
+    margin-top: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 50px;
```

Comment:
![image](https://user-images.githubusercontent.com/37579660/88346802-29453500-cd84-11ea-96c0-9bfb6207833e.png)
```suggestion
    margin: 50px auto;
```

이렇게 한번에 표현 할 수 있네요!

저도 잘 몰랐던 부분인데 그니 덕에 공부까지 했습니다~